utils/migrations.js

```diff
@@ -0,0 +1,347 @@
+String.prototype.addTab = function() {
+  return `${this.valueOf()}\t`;
+};
+
+String.prototype.addEndLine = function(semicolon = true) {
```

Comment:
This is confusing. Use a method to add the comma and another one to add the new line.